src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeStore.targets

```diff
@@ -220,15 +220,18 @@ Copyright (c) .NET Foundation. All rights reserved.
       <SkipUnchangedFiles Condition="'$(SkipUnchangedFiles)' == ''">true</SkipUnchangedFiles>
     </PropertyGroup>
 
-   <NETSdkError Condition="'2.0' > '$(_TargetFrameworkVersionWithoutV)'"
-                 ResourceName="NU1008"
-                 FormatArguments="$(TargetFrameworkMoniker)"/>
-
-    <NETSdkError Condition="'$(RuntimeIdentifier)' =='' and '$(_PureManagedAssets)' == ''"
-                 ResourceName="RuntimeIdentifierWasNotSpecified"/>
-
-    <NETSdkError Condition="'$(_TFM)' ==''"
-                 ResourceName="AtLeastOneTargetFrameworkMustBeSpecified"/>
+   <NETSdkMessage Condition="'2.0' > '$(_TargetFrameworkVersionWithoutV)'"
```

Comment:
There appear to be a bunch of whitespace-only diffs here. Please revert and amend.

Reply:
Nevermind.